DeusEx/Classes/DXREntranceRando.uc

```diff
@@ -3,9 +3,6 @@
 //=============================================================================
 class DXREntranceRando expands DXRBase;
 
-var int curMission;
-
-
 //Defines an edge of a map where you can transfer between maps
 //Via Teleporter or MapExit
 //In the main game, all of them have an inbound point and an outbound one
```

Comment:
(talking about the struct MapTransfer below this comment, and other structs)
I don't think the config keyword does anything inside of a struct

if you wanna go crazy, you could have an array of strings for what KeyIDs each map contains, and what KeyIDs each map requires/uses

you could also keep counters for how many multitools/lockpicks/passwords are found/used in each map, but there's a lot of overlap between needing keys, passwords, multitools, lockpicks...

I think a simple (and configurable?) banlist of MapTransfers would do the trick (for example, ban BatteryPark to Sewers), you could even reuse the struct MapTransfer for that, like

```
var config MapTransfer banlist[64];
```

Reply:
Yeah, I was already planning to add an "exitOnly" property to the MapTransfer structure, and so I think it would make sense to add the ban list to that structure rather than Connection.  My thought process is that MapTransfer holds the actual useful information about the exit/entry, so it could be more useful to attach that there.  Perhaps a "Heavily locked" property on the MapTransfer would convey the same info.